src/Chainweb/CutDB.hs

```diff
@@ -274,25 +275,37 @@ processCuts
     -> TVar Cut
     -> IO ()
 processCuts logFun headerStore payloadStore queue cutVar = queueToStream
+    & S.chain (\_ -> logFun @T.Text Info "start processing new cut")
```

Comment:
How much spam is this likely to produce?

Reply:
I wish it would produce a lot :-). Unfortunately, this is where the testnet nodes stall. So, I put in more logging here. I'll remove it again once Testnet is stable.

Overall, it doesn't produce too much logs. Generally, logging isn't a concern at this point. A concern might be how much data we store in ES, but that should be taken care of by proper configuration of ES. As long as logging doesn't slow down the Chainweb-nodes themself too much, we shouldn't care.